### ISSUE TITLE
`silk/lib/images/ui-bg_highlight-soft_75_cccccc_1x100.png` file not found while running collectstatic command

### ISSUE THREAD

**Issue by imtipu** [jazzband/django-silk#755]
While running command `python manage.py collectstatic --no-input` I got this error. Need to fix it `silk\lib\jquery-ui-1.13.2.min.css` file to update the path of the image
```
The CSS file 'silk\lib\jquery-ui-1.13.2.min.css' references a file which could not be found:
  silk/lib/images/ui-bg_highlight-soft_75_cccccc_1x100.png
```

**Comment by 2victor2** [comment]
I'm also getting this error in 5.3.1 version, but in 5.2.0 it works fine

**Comment by albertyw** [comment]
Fixed in #757 
Will be included in the next release
